metatron-app/metatron-discovery
# #3606 added setting grid column

### Description

There were many opinions that it was inconvenient to use because there is no setting for each column of the grid.

**Related Issue** : #3606 


### How Has This Been Tested?

1. Move to the grid chart creation screen.
2. Set the measurement to the pivot.
3. Change the format in the pivot's detail menu.
4. Check whether the format is applied to the column of the measurement value.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
